### ISSUE TITLE
isis2std 8 bit tiff output not correct range

### ISSUE THREAD

**Issue by scsides** [DOI-USGS/ISIS3#4840]
**ISIS version(s) affected**: 3.10.2

**Description**  
When I convert a 32-bit isis file to tiff 8-bit the tif file values
range from 6 to 255, not 0 to 255.

I used stretch to first put the pixel values in the range 0-255. I can
look at the stretch output and see the zero values. When I look at the
output tif file from isis2std what was 0 is now 6. I suspect there is a
reservation system for special pixels (but that would only require 0-4).

in stretch I tried adding null=0 lis=0 lrs=0 his=255 hrs=255 to the
call, still getting 6 for zero.

I have tried this on numerous image files.
**How to reproduce**  
isis2std format=tiff stretch=manual minimum=0 maximum=255

stretch from=M1397979727_LRmos.warp.cub
to=M1397979727_LRmos.warp.8bit.cub readfile=yes inputfile=stretch01.txt
null=0 lis=0 lrs=0 his=255 hrs=255
isis2std from=M1397979727_LRmos.warp.8bit.cub
to=M1397979727_LRmos.warp.str01.tif stretch=manual format=tiff minimum=0
maximum=255
**Possible Solution**  

**Additional context**

**Comment by Kelvinrr** [comment]
I dont think I was able to replicate? 
```
(isisdev) krodriguez@prog28:~/repos/isisbuild$ isis2std from=/tmp/8bit.cub to=M1397979727_LRmos.warp.str01.tif stretch=manual format=tiff minimum=0 maximum=255
Group = Results
  OutputFileName = M1397979727_LRmos.warp.str01.tif
  InputMinimum   = 0.0
  InputMaximum   = 255.0
End_Group
```

but I am also missing stretch01.txt, can you provide that?

**Comment by Kelvinrr** [comment]
So using ISIS 5.0, with the command `isis2std from=/work/users/ssides/bugs/isis2std/M1397979727_LRmos.warp.cub to=/tmp/8bit.tif format=tif stretch=manual minimum=0.06 maximum=0.15` (skipping the stretch as it's hard to reproduce that step without the stretch file and isis2std can do the stretch in one step), I get a DN range of 0 to 255. 

I'm inclined to think either 1. There is a problem with the two step approach of using stretch then re-stretching in isis2std, or 2. whatever the issue was has been fixed since ISIS 3.10.2.

**Comment by Kelvinrr** [comment]
Using the same command with ISIS version 3.10.2 also gives correct results. I think this was simply a processing error. 

I think we can close this, if there is something in those stretch files that is important to test then I we can reopen it, but without those we can't really move forward and it seems `isis2std from=/work/users/ssides/bugs/isis2std/M1397979727_LRmos.warp.cub to=/tmp/8bit.tif format=tif stretch=manual minimum=0.06 maximum=0.15` accomplished the task in a way that requires less processing.
